### DISCUSSION TITLE
Using this to feed custom website

### DISCUSSION THREAD

**Discussion by andy-blum** [cwendt94/espn-api#181] [Q&A] [answered]
I'm looking to build a tool to do some more advanced roster management than what ESPN allows (practice squad, player contract/salary, etc). How can I use this tool to feed data into my website?

**Comment by cwendt94** [comment, marked as answer]
That's a great question. This package will essentially allow you to get all of your League data programmatically, including which players are on free agency. From there you can try to build a more advance roster management with that information. However, I can see it being difficult as the package will always reflect your Leagues current data. You would need to store and manage your new advance roster on its own.

**Reply by andy-blum** [reply]
Will this export that data in a web-friendly format like JSON?

**Reply by cwendt94** [reply]
Not at the moment, right now all of the raw ESPN API data is wrapped into Classes. The [Wiki](https://github.com/cwendt94/espn-api/wiki) shows functionality and the data you can access.

At some point (when I have some extra time) I am planning on exposing the internal ESPN request wrapper that I created and used through out this package to make the API requests. This would allow users if they want, to get the raw JSON data from ESPN's API. I still need to make it more user friendly and write documentation for the request module. Here is the majority of the ESPN request code if you want to take a look https://github.com/cwendt94/espn-api/blob/master/espn_api/requests/espn_requests.py
